ios/.gitignore

```diff
@@ -1,2 +1,5 @@
-# XcodeGen
-*.xcodeproj
\ No newline at end of file
+.DS_Store
+/.build
+/Packages
+/*.xcodeproj
```

Comment:
maybe `ios/DroidKaigi 2021/Debug.xcodeproj` &  `ios/DroidKaigi 2021/Release.xcodeproj` is not ignored ?
Or is this line not needed?

Reply:
> ios/DroidKaigi 2021/Debug.xcodeproj & ios/DroidKaigi 2021/Release.xcodeproj is not ignored ?

Yes. these are ignored.
This ignore line is for Root `*.xcodeproj`  such as `DroidKaigi2021.xcodeproj` generated by `swift package generate-xcodeproj` command.